core-bundle/src/Resources/public/mootao.js

```diff
@@ -118,6 +118,10 @@ Tips.Contao = new Class(
 	options: {
 		id: 'tip',
 		onShow: function() {
+			var title = this.tip.getElement('div.tip-title');
+			if (!title || !title.innerHTML) {
+				return;
+			}
```

Comment:
Maybe we could even hide the tooltip if `event.page[y] == 1` in the `position` method, because it seems to be an error if a tooltip should be show on position 0